src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String HEADERS = "fruit,quantity";
+
+    @Override
+    public String getReport() {
+        return Storage.getStorage().entrySet().stream()
+                .map(totalFruits -> totalFruits.getKey() + "," + totalFruits.getValue()
+                        + System.lineSeparator())
+                .collect(Collectors.joining());
+    }
+
+    @Override
+    public void writeHeaders(BufferedWriter bufferedWriter) {
```

Comment:
do you use this method in code? where do you pass BufferedWriter to this method?

let's just add headers in getReport() method?